clouddq/classes/dq_rule_binding.py

```diff
@@ -348,6 +359,7 @@ def resolve_all_configs_to_dict(
                     "column_id": self.column_id,
                     "reference_columns_id": self.reference_columns_id,
                     "include_reference_columns": include_reference_columns,
+                    "include_all_reference_columns": include_all_reference_columns,
```

Comment:
Where is this used?

Reply:
@nauh I have added this in configs dictionary to identify the configs for job if it required all columns or no with this flag.